java/src/main/java/ai/rapids/cudf/DataType.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ai.rapids.cudf;
+
+import java.io.IOException;
+
+public class DataType {
+
+  DType typeId;
+  int scale;
+   public DataType(DType id) {typeId = id; }
+
+   public DataType(DType id,  int fp_scale) {
```

Comment:
Generally we use `camelCase` instead of `snake_case` for variable names in java.

Reply:
Sorry, missed this. Was switching from cpp.